5. Administrator Guides/Integrations/Zapier/README.md

```diff
@@ -11,4 +11,36 @@ Okay okay, before you get too excited there are a few things that we have to do
 6. Change `Redirect URI` to be `https://zapier.com/dashboard/auth/oauth/return/App32270API/`
 7. Click `Save Changes` then verify the save was successful.
 
-Did you complete everything? Yes? Good! [Here is your invite (hint: click me)](https://zapier.com/dashboard/auth/oauth/return/App32270API/)!
\ No newline at end of file
+Did you complete everything? Yes? Good! [Here is your invite (hint: click me)](https://zapier.com/dashboard/auth/oauth/return/App32270API/)!
```

Comment:
`https://zapier.com/developer/invite/32270/7c3feadc825db6ae9023ea2983e88875/` there's the correct link for the invite, if you want to update it.